aten/src/ATen/native/native_functions.yaml

```diff
@@ -5774,10 +5774,11 @@
     NestedTensorCPU, NestedTensorCUDA: _nested_tensor_strides
   autogen: _nested_tensor_strides.out
 
-- func: _nested_tensor_offsets(Tensor self) -> int[]
+- func: _nested_tensor_storage_offsets(Tensor self) -> Tensor
```

Comment:
Changing the name of a native function worries me

Reply:
As discussed, we had a manual exception added to the gen code before so this wasn't exposed to Python until now.

Unclear whether the FC / BC check will fail due to the name change (it shouldn't, but idk if it's aware of the gen code exception). Guess we'll see